tensorflow_datasets/core/visualization.py

```diff
@@ -85,12 +96,37 @@ def show_examples(ds_info, ds, rows=3, cols=3, plot_scale=3., image_key=None):
 
     image_key = image_keys[0]
 
-  label_keys = [
+  sequence_keys = [
```

Comment:
This functions starts having too many conditions/variables. Making the code harder to read. I think it would make sense to refactor it in smaller modules/functions. For instance:

```python
def show_examples(ds_info, ds, **options_kwargs):
  for visualizer in _ALL_VISUALIZERS:
    if visualizer.match(ds_info, ds):
      return visualizer.build(ds_info, ds, **options_kwargs)
  else:
    raise ValueError(f"Visualisation not supported for dataset `{ds_info.name}`")
```